app/graphql/types/change_visibility_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Types
+  class ChangeVisibilityType < Types::BaseObject
+    field :id, Integer, null: true
```

Comment:
id is woanders `ID`, hier `Integer`

> field :id, ID, null: true

Grund?

Reply:
ui... so gut kenne ich nicht mit dem GraphQl nicht aus... meinte also ID wäre besser?

Reply:
besser weiß nich, aber wäre einheitlicher zu den anderen types

Reply:
angepasst